### PR TITLE
Fixing broken links and typos in README. Closes #35

### LINKED ISSUE #35: Broken image tag in README.md
Currently the Imperial logo doesn't show up.

It's only a small thing, but first impressions count for users :wink:

### PR DESCRIPTION
**Pull request by davidorme**
I've disabled some markdownlint checks on the image lines (overlong and inline html). Controlling image sizes is a pain in markdown flavours, so I've used HTML as the more robust option.

Closes #35

### CODE REVIEW

**Comment by alexdewar** [comment]
I'd usually put the "closes #35" in the body of your message rather than the title.

**Comment by alexdewar** [comment]
It looks good to me and normally I would approve this, but I'll leave it to @robewers01 to do a review so he can have the practice :smile:

File: README.md [hunk 1]
```diff
@@ -19,12 +18,13 @@ Foundation to Professor Robert Ewers:
 * [NOMIS Award details](https://nomisfoundation.ch/people/robert-ewers/)
 * [NOMIS project summary](https://nomisfoundation.ch/research-projects/a-virtual-rainforest-for-understanding-the-stability-resilience-and-sustainability-of-complex-ecosystems/)
 
-[<img alt="NOMIS logo" src="https://nomisfoundation.ch/wp-content/uploads/2020/05/nomis-foundation-logo-372-by-151.png" width=250>](https://nomisfoundation.ch)
+<!-- markdownlint-disable-next-line  MD033 MD013-->
+[<img alt="NOMIS logo" src="docs/source/_static/images/logo-nomis-822-by-321.png" width=250>](https://nomisfoundation.ch)
```
Comment (robewers01) [inline README.md:22]: Just a comment for me to check I can make comments. Looks okay to me.

Comment (dalonsoa) [reply README.md:22]: It's ok, I cna see you :)

**Review by robewers01 — APPROVED** [review]

**Comment by alexdewar** [comment]
@davidorme This has been approved, so you can merge it now!

**Comment by alexdewar** [comment]
Ha, looks like you did it as I was typing my message :laughing:

**Comment by davidorme** [comment]
Wasn't sure if the approver or requester does the merge :-)

**Comment by davidorme** [comment]
I'll work on #32 now.

**Comment by alexdewar** [comment]
Usually it's the requester. As a reviewer you signal that you're happy with it by approving the PR.

**Pull request merged**